org.rncbc.qpwgraph.yml

```diff
@@ -0,0 +1,23 @@
+app-id: org.rncbc.qpwgraph
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+command: qpwgraph
+finish-args:
+  - "--device=dri"
+  - "--filesystem=xdg-run/pipewire-0"
+  - "--share=ipc"
+  - "--socket=x11"
```

Comment:
```suggestion
  - "--socket=fallback-x11"
```

Reply:
With this I have:
```
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vkkhrdisplay, vnc, wayland-egl, wayland, xcb
```

I've seen it elsewhere but I don't understand where the issue is 😕.

Reply:
then remove the wayland socket instead as it seems to be forcing xcb QPA

Reply:
See:
``` c++
#if defined(Q_OS_LINUX) && !defined(CONFIG_WAYLAND)
	::setenv("QT_QPA_PLATFORM", "xcb", 0);
#endif

```

Not sure why but that's upstream